### ISSUE TITLE
AssertionError

### ISSUE THREAD

**Issue by dmiroff** [matterport/Mask_RCNN#799]
I am running inspect_balloon_model on my machine and got assertion error at 1.b RPN Predictions:
For me it looks close to issue #23 , but solution from it doesn't help. nms_node is empty. 
There could be difference in tf versions. My setup: tf 1.9, keras 2.1.6, cuda 9.2, nvidia driver 396.37, Ubuntu 16.04. The only one change in the model by me was adding that 'print' on the screenshot.

![default](https://user-images.githubusercontent.com/31459590/42994011-a3c88384-8c15-11e8-843c-465c8e47c40a.png)

Hope, you could give me an advice.
Thanks in advance

**Comment by dmiroff** [comment]
The issue was solved by rollback of tf to 1.8 and cuda to 9.0.

**Comment by benediktbrandt** [comment]
In case any else comes accross this: In  tensorflow 1.9 it is now called NonMaxSuppressionV3:0, so the code should be:

```
nms_node = model.ancestor(pillar, "ROI/rpn_non_max_suppression:0")
if nms_node is None:
    nms_node = model.ancestor(pillar, "ROI/rpn_non_max_suppression/NonMaxSuppressionV2:0")
if nms_node is None:
    nms_node = model.ancestor(pillar, "ROI/rpn_non_max_suppression/NonMaxSuppressionV3:0")
```
